search/src/main/kotlin/com/alfresco/content/search/ListViewFilterChips.kt

```diff
@@ -54,12 +54,20 @@ class ListViewFilterChips @JvmOverloads constructor(
     }
 
     private fun String.wrapWithLimit(limit: Int, delimiter: String? = null, multipleValue: Boolean = false): String {
-        if (this.length <= limit)
+        if (this.length <= limit && delimiter == null)
             return this
 
-        if (delimiter != null && this.contains(delimiter)) {
-            val splitStringArray = this.split(delimiter)
-            return context.getString(R.string.name_truncate_in_end, splitStringArray[0].wrapWithLimit(20, ",", true), splitStringArray.size.minus(1))
+        if (delimiter != null) {
```

Comment:
If delimiter == null is checked in line 57, why we are again checking the same in line 60?

Reply:
line no. 57 checks the limit as well

Reply:
I think that Ankit is right we don't need line 61 check. above we have and condition, for sure with null value, it returns and never come down to execute the statement.